README.md

```diff
@@ -21,8 +21,7 @@ The framework is easy to adjust to meet your needs, because Yii has been designe
 Installation
 ------------
 
-- The minimum required PHP version of Yii is PHP 5.4.
-- It works best with PHP 7.
+- The minimum required PHP version of Yii is PHP 7.1.
```

Comment:
should probably state the version: `The minimum required PHP version of Yii 2.1 is PHP 7.1.`